addons/project/static/src/css/project.css

```diff
@@ -47,3 +47,12 @@
   margin-top: 6px;
   margin-right: 8px;
 }
+.oe_kanban_quick_create_arrow:before {
+ left: 280px;
+ top: 80px;
+}
+
+.oe_kanban_quick_create_margin {
+ margin-left: 180px !important;
+}
+
```

Comment:
Hello,
Custom CSS add for change position of arrow and text of menutip in task kanban, may be without custom css it is not possible but if there is any alternative solution then suggest me.
Thank you.